include/drivers/sensor/ina219.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2021 Leonard Pollak
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief Extended public API for the Texas Instruments INA219
+ */
+
+#ifndef ZEPHYR_INCLUDE_DRIVERS_SENSOR_INA219_H_
+#define ZEPHYR_INCLUDE_DRIVERS_SENSOR_INA219_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <drivers/sensor.h>
+
+enum sensor_channel_ina219 {
+	/** Shunt Voltage in V **/
+	SENSOR_CHAN_V_SHUNT = SENSOR_CHAN_PRIV_START,
+	/** Bus Voltage in V **/
+	SENSOR_CHAN_BUS_VOLTAGE,
+	/** Power **/
+	SENSOR_CHAN_BUS_POWER,
+	/** Current **/
+	SENSOR_CHAN_BUS_CURRENT,
```

Comment:
would probably be a good idea to namespace these channels, ie, `SENSOR_CHAN_INA219_BUS_VOLTAGE`

Reply:
i prefixed the channels as you suggested and updated the relevant parts of the code